web/src/components/RequestDetails.vue

```diff
@@ -0,0 +1,139 @@
+                  @click="getRequestOutput(requestDetails.request_id)">
+                </v-btn>
+              </template>
+            </v-tooltip>
+          </template>
+        </v-snackbar>
+      </v-card-title>
+      <v-alert v-if="requestDetails.status === 'success'" type="success" prominent>
+          Request <strong>{{ requestDetails.request_id }}</strong> completed successfully.
+      </v-alert>
+      <v-alert v-else-if="requestDetails.status === 'running'" type="info" prominent>
+          Request <strong>{{ requestDetails.request_id }}</strong> has <strong>{{ requestDetails.task_count - requestDetails.successful_tasks - requestDetails.failed_tasks }}</strong> Tasks remaining.
+      </v-alert>
+      <v-alert v-else-if="requestDetails.status === 'completed_with_errors'" type="warning" prominent>
+          Request <strong>{{ requestDetails.request_id }}</strong> completed with <strong>{{ requestDetails.failed_tasks }}</strong> failed Tasks.
+      </v-alert>
+      <v-alert v-else type="error" prominent>
+          Request <strong>{{ requestDetails.request_id }}</strong> was not successful.
+      </v-alert>
+      <v-card>
+          <v-list density="compact" v-model:opened="openGroups">
+            <v-list-group value="ids">
+              <template v-slot:activator="{ props }">
+                <v-list-item v-bind="props" title="Associated IDs"></v-list-item>
+              </template>
+              <v-list-item title="Request ID:">
+                  {{ requestDetails.request_id }}
+              </v-list-item>
+              <v-list-item title="Evidence ID:">
+                  {{ requestDetails.evidence_id }}
+              </v-list-item>
+              <v-list-item title="Reason:">
+                  {{ requestDetails.reason }}
+              </v-list-item>
+            </v-list-group>
+            <v-list-group value="details">
+              <template v-slot:activator="{ props }">
+                <v-list-item v-bind="props" title="Processing Details"></v-list-item>
+              </template>
+              <v-list-item title="Evidence Name:">
+                  {{ requestDetails.evidence_name }}
+              </v-list-item>
+              <v-list-item title="Last Update:">
+                  {{ requestDetails.last_task_update_time }}
+              </v-list-item>
+              <v-list-item title="Requester:">
+                  {{ requestDetails.requester }}
+              </v-list-item>
+              <v-list-item title="Status:">
+                  {{ requestDetails.status }}
+              </v-list-item>
+              <v-list-item title="Total:">
+                  {{ requestDetails.task_count }} Tasks
+              </v-list-item>
+              <v-list-item title="Running:">
+                  {{ requestDetails.running_tasks }} Tasks
+              </v-list-item>
+              <v-list-item title="Queued:">
```

Comment:
There is a new request status "pending" that should be visible in the request details.

Reply:
Ack updated to Pending instead